public/index.html

```diff
@@ -182,7 +179,9 @@ <h2 class="home-section-title">Testimonials and Notables</h2>
             </blockquote>
           </section>
           <button class="middle">
-            <a href="/testimonials.html">Read More of Our Customer Comments</a>
+            <a href="/testimonials.html"
+              >What our Clients are Saying about Us</a
```

Comment:
odd to have the up and down casing on the first letter, but i know that's a copy paste from their request